README.md

```diff
@@ -1149,7 +1213,7 @@ Current the following 5 log levels are supported:
    `LogLevel.OFF` can be used to turn off all logging from the Skyflow SDK.
 
 **Note:**
-* The ranking of logging levels are as follows:  `DEBUG` < `INFO` < `WARN` < `ERROR`
```

Comment:
I left this order as-is, but it seems like it should go the other direction to me. @ram7muthu what do you think? DEBUG gives the most logs, ERROR gives the least.